package.json

```diff
@@ -10,17 +10,26 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.11",
+    "@react-navigation/bottom-tabs": "^6.3.2",
+    "@react-navigation/material-bottom-tabs": "^6.2.2",
```

Comment:
You seem to be only using @react-navigation/bottom-tabs, remove @react-navigation/material-bottom-tabs

Reply:
Removed this unnecessary package